### ISSUE TITLE
Calculation of  MCC should return 0 in cases where equation would yield Inf

### ISSUE THREAD

**Issue by cchwala** [pycomlink/pycomlink#56]
The [calculation of the MCC](https://github.com/pycomlink/pycomlink/blob/0eeb08838c8ff6a3bac084d36fbb9a55dc032bfd/pycomlink/validation/stats.py#L179)
```
matthews_correlation = ((N_tp*N_tn-N_fp+N_fn) /
                        np.sqrt((N_tp+N_fp)*(N_tp+N_fn) *
                                (N_tn+N_fp)*(N_tn+N_fn)))
```
returns `np.Inf` in case instances of one class are missing.

The [MCC function of `scikit-learn`](https://github.com/scikit-learn/scikit-learn/blob/1495f6924/sklearn/metrics/classification.py#L793) returns 0 in these cases, they just do not get `Inf` but `NaN`, probably because they do the calculation via covariances.

We should also return 0 in those cases. 

A test must also be added.

**Comment by maxmargraf** [comment]
I put together a test and a fix in [this PR](https://github.com/pycomlink/pycomlink/pull/57).

**Comment by cchwala** [comment]
closed in #58
